assets/messages.yml

```
@@ -903,6 +903,24 @@ messages:
         unless your bug report is an exploit or contains information about your username
         or server.{panel}"
       fillname: []
+  panel-removed-version:
+    - project:
+        - bds
+        - mc
+        - mcd
+        - mce
+        - mcl
+        - mcpe
+        - realms
+      name: Panel - Remove affected version from resolved ticket
```

Ah sorry, one more minor thing: It should probably say "Remove***d*** affected ..."
```suggestion
      name: Panel - Removed affected version from resolved ticket
```

Though this is not completely consistent with "Panel - Unmark private issue" where the name describes what the user has done (i.e. "unmark private issue") while here it describes what the staff member / Arisa has done. So maybe this should be changed:
```suggestion
      name: Panel - Added affected version to resolved ticket
```

I am not quite sure which one is better.